tests/foreman/ui/test_discoveredhost.py

```diff
@@ -172,6 +172,7 @@ def test_positive_pxe_less_with_dhcp_unattended(session, provisioning_env):
 
 
 @skip_if_bug_open('bugzilla', 1665471)
```

Comment:
`1728306` is closed as duplicate of `1658659` and `1658659` is closed as well.
Can we replace `1728306` with `1728306` ?

Reply:
Yeah, I wanted to remove all of them in one go/script. But I will remove it now. :)

Reply:
Resolved.